internal/backend/service.go

```diff
@@ -64,8 +64,9 @@ type DefaultService struct {
 // and the order as value.
 func requestStateOrder() map[api.Status]int {
 	return map[api.Status]int{
-		// empty is the default and assumed to be the same as requested
+		// empty is the default and assumed to be the same as initiated
 		"":                  0,
+		api.InitiatedStatus: 0,
 		api.RequestedStatus: 0,
 		api.GrantedStatus:   1,
 		api.DeniedStatus:    2,
```

Comment:
I think you might need to define ordering for expired

Reply:
It is already there

Reply:
pls ignore my comment, not relevant